src/components/Marketplace/ProduceCard.js

```diff
@@ -78,13 +88,17 @@ const useStyles = makeStyles({
 
 export default function ProduceCard(props) {
   const {
-    cropName, farmID, unitPrice, unitType,
+    // eslint-disable-next-line no-unused-vars
+    produceID, farmID, unitPrice, unitType, season, // (TODO: remove later when season is used)
   } = props;
   const [farmName, setFarmName] = useState('');
+  const [produceName, setProduceName] = useState('');
+  // eslint-disable-next-line no-unused-vars
+  const [produceCategory, setProduceCategory] = useState(''); // (TODO: remove later when used)
 
   // Get farm name from Airtable if produce has farm id linked to it
   useEffect(() => {
-    if (farmID === null) {
+    if (farmID === null || farmID[0].length < 5) {
```

Comment:
Why are we checking to see if it's less than 5?

Reply:
Because of the way that I pass farmID, it gets converted so the null check doesn't always work. I just used an arbitrary short length of 5 since I'm assuming none of the IDs are going to be that short if they're actually valid, but if there's a possibility that they might be shorter I can change the check.